src/io/p5.Table.js

```diff
@@ -1213,6 +1213,32 @@ p5.Table.prototype.getObject = function(headerColumn) {
  *
  * @method  getArray
  * @return {Array}
+ * @example
+ * <div class="no render">
```

Comment:
There shouldn't be a space between "no" and "render" otherwise they are actually two classes `.no` and `.render` instead of one class `.norender`.